pkg/ccl/backupccl/bench_covering_test.go

```diff
@@ -53,12 +57,21 @@ func BenchmarkRestoreEntryCover(b *testing.B) {
 						b.Run(fmt.Sprintf("numSpans=%d", numSpans), func(b *testing.B) {
 							ctx := context.Background()
 							backups := MockBackupChain(numBackups, numSpans, baseFiles, r)
+							latestIntrosByTable, err := backupinfo.FindLatestReIntroductionByTable(backups,
+								keys.TODOSQLCodec, hlc.Timestamp{})
```

Comment:
nit: can we make this the system tenant codec, we're trying to get rid of TODOSQLCodec so no point adding another?